subiquity/server/controllers/filesystem.py

```diff
@@ -1252,12 +1256,15 @@ async def v2_add_partition_POST(self, data: AddPartitionV2) -> StorageResponseV2
             requested_size = data.gap.size
         # empty string is an unformatted partition
         fstype = data.partition.format or None
-        spec = {
-            "size": requested_size,
-            "fstype": fstype,
-            "mount": data.partition.mount,
+        spec: FileSystemSpec = {
+            "size": requested_size,  # FIXME this looks unused, watch out!
```

Comment:
Uh yeah this can probably go, right? Score +1 for static analysis

Reply:
Right, I finally dropped the key instead of adding the FIXME